### PR TITLE
Tell Vim to forget what it thinks it knows about .tf files

### PR DESCRIPTION
**Pull request by dimbleby**

### CODE REVIEW

File: ftdetect/terraform.vim [hunk 1]
```diff
@@ -1,3 +1,5 @@
+" By default, Vim treats .tf files as TinyFugue - tell it not to.
```
Comment (sudoforge) [inline ftdetect/terraform.vim]: The reason for the change should go into the commit body -- that's what it's there for! :)

Locally, you can remove this line, and then execute the following commands to clean up the commit before it's merged:

* `git commit`[`-a`][doc:git-commit-a][`--amend`][doc:git-commit--amend]
* update the commit body with more details
  * make sure to leave a blank newline between the subject and body!
* `git push`[`--force`][doc:git-push--force]

[doc:git-commit-a]: https://git-scm.com/docs/git-commit#Documentation/git-commit.txt--a
[doc:git-commit--amend]: https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---amend
[doc:git-push--force]: https://git-scm.com/docs/git-commit#Documentation/git-push.txt---force

File: ftdetect/terraform.vim [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
+autocmd! filetypedetect BufRead,BufNewFile *.tf
```
Comment (sudoforge) [inline ftdetect/terraform.vim]: Is this issue only present with `*.tf` files, and none of the other extensions handled by this plugin?

Comment (dimbleby) [reply ftdetect/terraform.vim]: Looks like it - see [here](https://github.com/vim/vim/blob/26910de8b0da6abab87bd5a397330f9cbe483309/runtime/filetype.vim#L1679-L1680)

File: ftdetect/terraform.vim (hunk 1 above)
Comment (dimbleby) [reply ftdetect/terraform.vim]: This is an unusual thing to do in an ftplugin - I reckon that this comment is worth its weight.

On the other hand it's hardly worth arguing about; I'll make the change...

(I am familiar with git, thanks).

Comment (dimbleby) [reply ftdetect/terraform.vim]: I changed my mind, this *is* worth a comment and I'm putting it back!

Comment (sudoforge) [reply ftdetect/terraform.vim]: I suppose my preference for removing it stems from the fact that there are _no_ other comments within any of the `*.vim` files within this plugin; but I'm not a maintainer, so I'll leave it to them to review!

Comment (dimbleby) [reply ftdetect/terraform.vim]: > ... there are no other comments within any of the *.vim files within this plugin ...

That's just not true.  

Though for what it's worth, several of the existing comments do tend towards stating the obvious - it actually would make more sense to get rid of those than to get rid of this one.

Comment (sudoforge) [reply ftdetect/terraform.vim]: I could definitely be wrong; I just did a quick `rg '^#` and didn't get any results. My bad!

Comment (dimbleby) [reply ftdetect/terraform.vim]: Yeah, that's not how to write a comment in VimL...

Comment (sudoforge) [reply ftdetect/terraform.vim]: Hahaha, whoops, you're absolutely right. That's what I get for trying to be useful without thinking :)

Comment (sudoforge) [reply ftdetect/terraform.vim]: It's been "one of those days" for a few months now...

**Pull request merged**